service/controller/v2/context/context.go

```diff
@@ -0,0 +1,32 @@
+package context
```

Comment:
This package was renamed to `controllercontext` in aws-operator to avoid ambiguity: https://github.com/giantswarm/aws-operator/tree/master/service/controller/v14/controllercontext